src/component/Collection/CollectionDetailItemComponent.js

```diff
@@ -90,15 +135,22 @@ export default class CollectionDetailItemComponent extends Component {
               const tracks = trackList[theme['themeName']] || [];
               const processedTracks = tracks.map((track) => ({
                 ...track,
-                imageName: track.imageName.replace(/\s+/g, ''),
+                searchTrackName: track.trackName.replace(/\s+/g, ''),
               }));
-              const searchTrack = hansearch(processedTracks, mapInputValue.replace(/\s+/g, ''), ["imageName"]);
+              const searchTrack = hansearch(processedTracks, mapInputValue.replace(/\s+/g, ''), ["searchTrackName",'imageName','trackName']);       
+              console.log(searchTrack)       
               if (!tracks.length) {
                 return null;
               }
-              return searchTrack.items.map(track => {
+               return searchTrack.items.map(track => {
                   return (
-                    <div key={ `key-detail-${track['trackName']}` } className='collection-detail-track'>
```

Comment:
[Question]
Is there a reason to remove key? If there is no strong reason, I think we should keep key property.